### ISSUE TITLE
Reds used in links fails AA contrast requirements

### ISSUE THREAD

**Issue by Kataract** [CLEpy/clepy-website#18]
From aXe testing:
> Element has insufficient color contrast of 4.44 (foreground color: #d9411e, background color: #ffffff, font size: 10.5pt, font weight: normal). Expected contrast ratio of 4.5:1

This goes for instances of `<a>` as well as `<a class="btn">`.  Bumping the color from `#d9411e` to `#bb371a` ups the ratio to 5.72:1 which passes AA.

**Comment by mUtterberg** [comment]
Apparently they are @orange. X-/
Change: included!
